observability/telemetry/src/main/scala/com/daml/telemetry/OpenTelemetryOwner.scala

```diff
@@ -62,19 +66,44 @@ case class OpenTelemetryOwner(setAsGlobal: Boolean, reporter: Option[MetricsRepo
 
 object OpenTelemetryOwner {
 
-  def addViewsToProvider(builder: SdkMeterProviderBuilder): SdkMeterProviderBuilder = {
-    builder
+  def addViewsToProvider(
+      builder: SdkMeterProviderBuilder,
+      histograms: Seq[HistogramDefinition],
+  ): SdkMeterProviderBuilder = {
+    // Only one view is going to be applied, and it's in the order of it's definition
+    // therefore the config views must be registered first to be able to override the code defined views
+    val builderWithCustomViews = histograms.foldRight(builder) { case (histogram, builder) =>
+      builder.registerView(
+        histogramSelectorWithRegex(histogram.nameRegex),
+        explicitHistogramBucketsView(histogram.bucketBoundaries),
+      )
+    }
+    builderWithCustomViews
```

Comment:
I'm wondering if we have documented these pre-defined buckets anywhere outside of the code. If not, it may be useful to have these next to the specific metrics or on the [common metrics page](https://docs.daml.com/ops/common-metrics.html).